src/core/halide2isl.cc

```diff
@@ -185,7 +185,7 @@ isl::map extractAccess(
 
 std::pair<isl::union_map, isl::union_map>
 extractAccesses(isl::set domain, const Stmt& s, AccessMap* accesses) {
-  class FindAccesses : public IRVisitor {
+  class FindAccesses : public IRGraphVisitor {
     using IRVisitor::visit;
```

Comment:
Should be IRGraphVisitor here, and in the calls to IRVisitor::visit below.

Reply:
The function is virtual, so it should not matter :)